lib/usage.js

```diff
@@ -397,8 +397,11 @@ module.exports = function (yargs, y18n) {
 
   // guess the width of the console window, max-width 80.
   function windowWidth () {
-    const wsize = require('window-size')
-    return wsize.width ? Math.min(80, wsize.width) : null
+    if (process && process.stdout && process.stdout.columns && Math.min(80, process.stdout.colums)) {
```

Comment:
The last term in this conjunction seems a bit odd? (Also: `typeof process === 'object' is probably the better first check here, at least if there’s a chance this file will ever switch to strict mode?)